src/search-manager/FilterByPublished.tsx

```diff
@@ -62,7 +62,7 @@ const FilterByPublished = ({
       onChange={() => { toggleFilterMode(filter); }}
     >
       <div>
-        {filterLabels[filter]}
+        {filterLabels[filter]}{' '}
         <Badge variant="light" pill>{publishStatus[filter] ?? 0}</Badge>
```

Comment:
Actually, our translated filter labels _should_ be formatted text that includes a variable for this count badge, so that RTL languages can opt to put the count on the left side of the text label. But since that's out of scope for this change it's ok to do as a follow-up PR.

CC @ChrisChV

Reply:
Alternatively, this spacing can be handled with CSS.